packages/devil.rb

```diff
@@ -3,26 +3,21 @@
 class Devil < Package
   description 'Library for reading several different image formats'
   homepage 'http://openil.sourceforge.net/'
-  version '1.8.0-e342'
+  version '1.8.0-6f3d5e9'
   license 'LGPL-2.1'
-  compatibility 'all'
-  source_url 'https://github.com/DentonW/DevIL/archive/e34284a7e07763769f671a74b4fec718174ad862.zip'
-  source_sha256 '73728ff9f877b9247728ae46148866eea81092ccefb538a28640fe31b73d74ac'
+  compatibility 'x86_64'
+  source_url 'https://github.com/DentonW/DevIL.git'
+  git_hashtag '6f3d5e9bc5d173f56060fc8e14bed35c8edcfdcc'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/devil/1.8.0-e342_armv7l/devil-1.8.0-e342-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/devil/1.8.0-e342_armv7l/devil-1.8.0-e342-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/devil/1.8.0-e342_i686/devil-1.8.0-e342-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/devil/1.8.0-e342_x86_64/devil-1.8.0-e342-chromeos-x86_64.tar.xz'
+    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/devil/1.8.0-6f3d5e9_x86_64/devil-1.8.0-6f3d5e9-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '7d45e28698eb99159d98b97760d11ddecff0ac87412b7cf249fb7e188b320502',
-     armv7l: '7d45e28698eb99159d98b97760d11ddecff0ac87412b7cf249fb7e188b320502',
-       i686: 'c75491427f9a75e33b8bca9f33b24cef15afc869118298d41177d45d3a913f4b',
-     x86_64: '99de10866bec2ac80094d5f3ad8a4d906cb6597dd01a2cce7d408533488ce330'
+    x86_64: '494336d49fee91201c74574356a2bf5c1e5bdc6e5d15bcc03decb99210286664'
```

Comment:
Missing builds for arm and i686?

Reply:
Builds for those architectures are currently broken, as devil has not been updated to work with newer versions of jasper.

See https://github.com/DentonW/DevIL/pull/102